wherehows-web/app/components/dataset-aclaccess-users.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import { get } from '@ember/object';
+
+export default Component.extend({
```

Comment:
The es classes syntax is preferred here:

`export default DatasetAclUsers extends Componet { ... }`